# TimeDist.msg
time timestamp
float64 distanceLeft
float64 distanceRight
